OWNERS

```diff
@@ -4,6 +4,7 @@ approvers:
 - ixdy
```

Comment:
@ixdy , would you still like to be listed here, or move to emeritus_approvers?

Reply:
feel free to move me to emeritus_approvers.

Reply:
done